packages/xo-web/src/xo-app/vm/index.js

```diff
@@ -81,7 +81,8 @@ import VmActionBar from './action-bar'
       pool: getPool(state, props),
       srs: getSrs(state, props),
       vbds: getVbds(state, props),
-      vdis: getVdis(state, props),
+      // Workaround to get VDIs as a self user
+      vdis: getVdis(state, props, vm.resourceSet !== undefined),
```

Comment:
Should we fix it more generally (ie pass `true`) since that's also a problem for non-self users with ACLs on the VM?